wled00/FX.cpp

```diff
@@ -5890,14 +5878,14 @@ uint16_t mode_2Dcrazybees(void) {
 
   if (strip.now > SEGENV.step) {
     SEGENV.step = strip.now + (FRAMETIME * 16 / ((SEGMENT.speed>>4)+1));
-
-    SEGMENT.fadeToBlackBy(32);
-
+    SEGMENT.fadeToBlackBy(32 + ((SEGMENT.check1*SEGMENT.intensity) >> 2));
+    SEGMENT.blur(SEGMENT.intensity >> 1, SEGMENT.check1);
     for (size_t i = 0; i < n; i++) {
-      SEGMENT.addPixelColorXY(bee[i].aimX + 1, bee[i].aimY, CHSV(bee[i].hue, 255, 255));
-      SEGMENT.addPixelColorXY(bee[i].aimX, bee[i].aimY + 1, CHSV(bee[i].hue, 255, 255));
-      SEGMENT.addPixelColorXY(bee[i].aimX - 1, bee[i].aimY, CHSV(bee[i].hue, 255, 255));
-      SEGMENT.addPixelColorXY(bee[i].aimX, bee[i].aimY - 1, CHSV(bee[i].hue, 255, 255));
+      CRGB flowerCcolor = ColorFromPalette(SEGPALETTE,bee[i].hue);
```

Comment:
Why not use `SEGMENT.color_from_palette()` to keep using WLED native pixels and behaviour?

Reply:
Regarding `ColorFromPalette()` and `SEGMENT.color_from_palette()`.

The former was used in FastLED effects ported to WLED with minimal effort (or to keep changes simple). Unfortunately this does not adhere to WLED's "default" behaviour for palettes and color, unless specifically crafted within effect. So I would prefer to use `SEGMENT.color_from_palette()` wherever possible if there are changes to an effect function.

Reply:
generally agreed. 
`color_from_palette` does some additional checks and calculations, plus converts to 32bits, then FX convert it back to CRGB, then setPixelColor converts it again to 32bit. That makes FX slow that call this for every pixel so I don't see it as a good solution. I have some basic ideas on how to fix this with CRGBW (basically overload some functions to accept RGB and RGBW and calculate the fastest way possible)

Reply:
> then FX _may_ convert it back to CRGB,

This (and other manipulations like that) will need to be removed or converted. `color_from_palette()` is there for a reason, to ensure consistent behaviour of "Default" options.